platform/web/doc_classes/EditorExportPlatformWeb.xml

```diff
@@ -86,6 +86,12 @@
 			If [code]true[/code], the exported game will support threads. It requires [url=https://web.dev/articles/coop-coep]a "cross-origin isolated" website[/url], which may be difficult to set up and is limited for security reasons (such as not being able to communicate with third-party websites).
 			If [code]false[/code], the exported game will not support threads. As a result, it is more prone to performance and audio issues, but will only require to be run on an HTTPS website.
 		</member>
+		<member name="variant/emscripten_pool_size" type="int" setter="" getter="">
+			TODO
+		</member>
+		<member name="variant/godot_pool_size" type="int" setter="" getter="">
+			TODO
```

Comment:
Something like "Override for the default size of the WorkerThreadPool. This setting is used when the WorkerThreadPool size is set to -1 (which it is by default). This size must be smaller than [member variant/emscripten_pool_size] otherwise deadlocks may occur. When using threads this size needs to be large enough to accommodate features that rely on having a dedicated thread like [member ProjectSettings.physics/2d/run_on_separate_thread] or [member ProjectSettings.rendering/driver/threads/thread_model]. In general, it is best to ensure that this is at least 4 and is at least at least 2 or three less than [member variant/emscripten_pool_size]."

@Faless How does that sound?

Reply:
This sounds good, the only it's not clear is what you mean by:
```
This setting is used when the WorkerThreadPool size is set to -1 (which it is by default).
```
Is that a setting? Or a method? We should probably reference it in case.